src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Products/Blocks/Tab/basic_settings_tab.html.twig

```diff
@@ -0,0 +1,136 @@
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+<div role="tabpanel" class="form-contenttab tab-pane active" id="product_basic_settings_tab_content">
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-md-9">
+
+        <div class="summary-description-container">
+          <ul class="nav nav-tabs bordered">
+            <li id="tab_description_short" class="nav-item"><a href="#description_short" data-toggle="tab" class="nav-link description-tab active">{{ 'Summary'|trans({}, 'Admin.Catalog.Feature') }}</a></li>
+            <li id="tab_description" class="nav-item"><a href="#description" data-toggle="tab" class="nav-link description-tab">{{ 'Description'|trans({}, 'Admin.Global') }}</a></li>
+          </ul>
+
+          <div class="tab-content">
+            <div class="tab-pane panel panel-default active" id="description_short">
+              {{ form_widget(productForm.basic_settings.short_description) }}
+            </div>
+            <div class="tab-pane panel panel-default" id="description">
+              {{ form_widget(productForm.basic_settings.description) }}
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="col-md-3">
+        {% if isCombinationFeatureActive %}
+          <div class="form-group mb-3" id="product_with_combinations_selection_block">
+            <h2>
+              {{ 'Combinations'|trans({}, 'Admin.Catalog.Feature') }}
+              <span class="help-box"
+                    data-toggle="popover"
+                    data-content="{{ 'Combinations are the different variations of a product, with attributes like its size, weight or color taking different values. Does your product require combinations?'|trans({}, 'Admin.Catalog.Help') }}">
+              </span>
+            </h2>
+            {{ form_widget(productForm.basic_settings.with_combinations) }}
+            <div id="product_type_combinations_shortcut">
+                <span class="small font-secondary">
+                  {{ "Advanced settings in [1][2]Combinations[/1]"|trans({}, 'Admin.Catalog.Help')|replace({'[1]': '<a href="#tab-step3" onclick="$(\'a[href=\\\'#step3\\\']\').tab(\'show\');" class="btn sensitive px-0">', '[/1]': '</a>', '[2]': '<i class="material-icons">open_in_new</i>'})|raw }}
+                </span>
+            </div>
+          </div>
+        {% endif %}
+
+        <div class="form-group mb-4">
+          <h2>
+            {{ 'Reference'|trans({}, 'Admin.Catalog.Feature') }}
+            <span class="help-box"
+                  data-toggle="popover"
+                  data-content="{{ 'Your reference code for this product. Allowed special characters: .-_#\.'|trans({}, 'Admin.Catalog.Help') }}" >
```

Comment:
Did you add the "\" character? I have this wording currently: _Your reference code for this product. Allowed special characters: .-_#._

Reply:
It's just closes a string, if I understand you correctly. :grin:

Reply:
Sorry @sarjon, I misspelled the comment, I was talking about the " \ " character. You can compare with the current wording used.

Reply:
```suggestion
                  data-content="{{ 'Your reference code for this product. Allowed special characters: .-_#.'|trans({}, 'Admin.Catalog.Help') }}" >
```